src/core/product-transformer.ts

```diff
@@ -71,6 +71,35 @@ namespace Utils {
       return Utils.remap(transformed, effectiveStructure, baseProduct);
     };
   }
+
+  export const transformData = (data: any) => {
```

Comment:
Why not combine this function with transformPrice since it is just basically a one-liner. Also you could name it transformPrice instead. TransformData is probably not intuitive.

Reply:
I did transformData so that it would look more intuitive when passing in data (the use case here). Also, this way it would be easier to extend transformData in the future if other fields need to be extended.